rhc-ose-ansible/roles/secure-registry/tasks/main.yaml

```diff
@@ -0,0 +1,164 @@
+---
+
+  - name: "Check that Openshift Docker Registry exists"
+    command: "{{ openshift.common.client_binary }} get deploymentConfig {{ registry_dc }} -n {{ openshift_registry_project }}"
+    register: registry_exists
+    delegate_to: "{{ groups['masters'][0] }}"
```

Comment:
I don't like the idea of using named host groups. It should be up to the play to declare which host groups this role is applied to. I would rather us use `{{ play_hosts[0]}}`